tests/system/alerts/test_alerts.py

```diff
@@ -84,6 +85,58 @@ def test_job_failure_alert(self):
         self._validate_notifications_on_nuclio(
             nuclio_function_url, expected_notifications
         )
+        # get all activations
+        activations = self._run_db.list_alert_activations(
+            project=self.project_name,
+            name=alert_name,
+            severity=[
+                alert_objects.AlertSeverity.LOW,
+                alert_objects.AlertSeverity.HIGH,
+            ],
+            entity=f"~{run_id}*",
+            entity_kind=alert_objects.EventEntityKind.JOB,
+            event_kind=alert_objects.EventKind.FAILED,
+        )
+        assert len(activations) == 2
```

Comment:
why do we have two activations here ? 
according to the test the alert should be triggered once